### ISSUE TITLE
As a user I want to see on the request page a count of relevant PRs and commits

### ISSUE THREAD

**Issue by kohshunyuan** [FundRequest/platform#397]
### Expected behaviour
Each request on this page should show a count of relevant PRs and commits
https://fundrequest.io/requests

### Actual behaviour
This information is not available

### Steps to reproduce the behaviour
- None

**Comment by karelstriegel** [comment]
Is the reason for these metrics to understand if and how many people are working on a specific issue?

**Comment by kohshunyuan** [comment]
It's more for the user to be able to prioritize which issue they want to investigate. From my own experience, when I first looked at the issues list, I found some issues that was interesting and clicked in to look at the descriptions and thought about potential causes/fixes. But while looking through the github history I realised that someone had already pushed a fix tagged to those issues. Ideally, I would be made aware of this right at the request list page and I could have spent that time/effort looking at other unsolved issues.

**Comment by karelstriegel** [comment]
In the future we plan to add staking if you start working on the issue, so should other users that you have picked up the request. Would that satisfy your need?

Having the information that you describe on the overview page would mean that we have to query the api of github for each issue, that would degrade performance.

**Comment by kohshunyuan** [comment]
Yes that would work, as long as there's some indication that _someone_ is looking at it (maybe even show how many users are staking on one request).
